src/models/Token.js

```diff
@@ -0,0 +1,13 @@
+import { DataTypes } from 'sequelize';
+import { sequelize } from '../utils/db.js';
+import { User } from './User.js';
+
+export const Token = sequelize.define('token', {
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+});
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
Add [empty line in the end of file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)